pulseapi/profiles/migrations/0025_transfer_user_isactive_to_profile.py

```diff
@@ -0,0 +1,33 @@
+from django.db import migrations
+from django.conf import settings
+
+
+def match_profile_isactive_to_user_isactive(apps, schema):
+    EmailUser = apps.get_model('users', 'EmailUser')
+    UserProfile = apps.get_model('profiles', 'UserProfile')
+
+    network_pulse_users = EmailUser.objects.all()
+
+    for user in network_pulse_users:
+
+        users_profile, created = UserProfile.objects.get_or_create(related_user=user)
```

Comment:
Since there are still around 4 user accounts without linked profiles in the DB (bug we are aware of and going to create a ticket to investigate), I needed to use the "get_or_create" method so the migration doesnt return an error when trying to find a users profile.